app/lib/package/backend.dart

```diff
@@ -175,6 +175,12 @@ class PackageBackend {
         .cast();
   }
 
+  /// Returns the latest releases info of a package.
+  Future<LatestReleases> latestReleases(Package package) async {
+    // TODO: implement runtimeVersion-specific release calculation
+    return package.latestReleases;
```

Comment:
So you're thinking we'll start storing `LatestReleases` as a runtimeVersioned entity? or...

I'm mostly wondering if we should make a property on the `Package` entity that is `runtimeVersioned` or a new entity that is runtimeVerisoned..

The new entity is certainly something we're already pretty skilled at doing.

Reply:
> So you're thinking we'll start storing LatestReleases as a runtimeVersioned entity? or...

I think last time we've discussed this, we've considered using a map on the `Package` like this:
```
{
  '2021.02.02': {'stable': '1.0.0', 'preview': '2.0.0'},
  '2021.03.25': {'stable': '2.0.0'}
}
```

However, I think the separate entity has better isolation between releases, and may be worth doing it that way. I'd still use this new structure in both cases.

Reply:
Okay, I'm down with keeping it in a separate entity, so long as we make sure to cache it whenever we load it :D